### ISSUE TITLE
bug: Cannot implement Kotlin interface DgsReactiveCustomContextBuilderWithRequest in Java

### ISSUE THREAD

**Issue by petrmac** [Netflix/dgs-framework#2023]
Please read our [contributor guide](https://github.com/Netflix/dgs-framework/blob/master/CONTRIBUTING.md) before
creating an issue.

## Expected behavior
Interface should be possible to implement 

## Actual behavior
When trying to implement the interface in Java project, the interface is not visible:
```
error: cannot find symbol
public class CredentialsContextBuilder implements DgsReactiveCustomContextBuilderWithRequest<CredentialsContext> {
                                                  ^
  symbol: class DgsReactiveCustomContextBuilderWithRequest
```

## Steps to reproduce

Implement the interface in Java project (not Kotlin). 
Using the version: `graphql-dgs-reactive-9.1.0.jar`

Initially was using: 
`implementation "com.netflix.graphql.dgs:graphql-dgs-webflux-starter"`
but now migrated to the newly endorsed  
`implementation "com.netflix.graphql.dgs:graphql-dgs-spring-graphql-starter"`

Any help would be greatly appreciated!

**Comment by awesor** [comment]
This was an awesome fix!  Thank you so much for getting on this, @asibross so quickly.  

Out of curiosity, since we're really eager on my end to use this, when are we looking at cutting a new release (or rc) that would contain this fix?

**Comment by paulbakker** [comment]
Released in 9.2.0.
